build.gradle.kts

```diff
@@ -23,7 +23,7 @@ kotlin {
             kotlin.srcDirs(tasks.yamlTestSuiteDataSources)
 
             dependencies {
-                implementation(platform("io.kotest:kotest-bom:5.8.0"))
+                implementation(project.dependencies.platform("io.kotest:kotest-bom:5.8.0"))
```

Comment:
Why this change? The former was more compact.

Reply:
It was deprecated

https://github.com/JetBrains/kotlin/blob/v1.9.21/libraries/tools/kotlin-gradle-plugin-api/src/common/kotlin/org/jetbrains/kotlin/gradle/plugin/HasKotlinDependencies.kt#L76-L81